### PR TITLE
Fix issue #15597 — Mention dropup variation in dropdown docs

### LINKED ISSUE #15597: .dropup should be mentioned in main Dropdown docs
It's not currently mentioned in http://getbootstrap.com/components/#dropdowns but ought to be.
Right now it's only documented in the context of button groups (http://getbootstrap.com/components/#btn-dropdowns-dropup).
Refs #15142.

### PR DESCRIPTION
**Pull request by kkirsche**
Fixes #15597. Mention dropup variation in dropdown docs

Other pull request (#15659) was so messed up I sincerely have no clue how to save it. The recommended command doesn't work, can't believe rebasing is so problematic...

### CODE REVIEW

**Comment by cvrebert** [comment]
CC: @mdo

**Comment by vsn4ik** [comment]
- Remove `<h3>Example</h3>`;
- Move `data-example-id=""` to `<h3>Dropup Variation</h3>` and change value;
- We must somehow keep the style of `.dropdown` (`position: static`), but I do not know how to do it.

**Comment by kkirsche** [comment]
I'm not sure I follow you @vsn4ik on point 3

**Comment by vsn4ik** [comment]
My mistake on point 2: `data-example-id` not need be moved. Only change, because `data-example-id="static-dropdown"` already exists.

File: docs/_includes/components/dropdowns.html [hunk 1]
```diff
@@ -33,6 +33,36 @@ <h3 id="dropdowns-example">Example</h3>
 </div>
 {% endhighlight %}
 
+  <h3 id="dropdowns-dropup">Dropup variation</h3>
+  <p>Trigger dropdown menus above elements by adding <code>.dropup</code> to the parent.</p>
+  <div class="bs-example" data-example-id="static-dropup">
+    <div class="dropup clearfix">
```
Comment (mdo) [inline docs/_includes/components/dropdowns.html]: Did I add the `clearfix` at some point, or is that new to this PR?

**Comment by mdo** [comment]
My only concern with this is that it's basically an exact dupe of another docs section.

**Comment by kkirsche** [comment]
I didn't add one to this. I copied and pasted to use what you had. 

I had started with a small note only, what would you be interested in as a solution?

**Comment by mdo** [comment]
Yeah, I think so @kkirsche.

**Comment by kkirsche** [comment]
@mdo What do you think about the following idea:

In the example section we add the following sentence:

```
Dropdown menus can also be triggered above elements by adding <code>.dropup</code> to the parent.
```

Then in the example section include one of each. This way, we reduce the space necessary by not including a standalone example for just the dropup variation.

The other idea I have would be to include the following sentence with no visible example:
```
Dropdown menus can also be triggered above elements, refer to <a href="components/#btn-dropdowns-dropup">Button group dropup variation</a> for more information and examples.
```

What is your preference?

**Comment by mdo** [comment]
A live example to go with it is probably preferable, so we can go with that.

**Comment by kkirsche** [comment]
Changed to the first option which uses short sentence with a live example in the first example box.

**Comment by cvrebert** [comment]
@mdo LGTY?

**Comment by mdo** [comment]
:shipit:

**Comment by kkirsche** [comment]
Thanks guys! :+1:

**Pull request merged**